osudroidresource/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -1,62 +1,157 @@
+
+	<string name="network_not_connected">Conexão com a internet falhou, por favor cheque as configurações de internet</string>
+	<string name="xml_parser_failed">Excessão na análise de dados</string>
+	<string name="io_exception_error">Excessão no fluxo de arquivos</string>
+	<string name="http_status_code_error">Excessão de rede, código do erro：%d</string>
+	<string name="http_exception_error">Excessões de rede, timeout solicitado</string>
+	<string name="socket_exception_error">Excessão de rede, tempo limite atingido</string>
+
+	<string name="binfoStr1" formatted="false">Duração: %s BPM: %s Combo: %d</string>
+	<string name="binfoStr2" formatted="false">Círculos: %d Sliders: %d Spinners: %d (MapId: %d)</string>
+
+	<!-- Crash -->
+	<string name="crash">Desculpa, situação anormal, fechando o jogo.</string>
+	<string name="app_run_code_error">Erro fatal encontrado</string>
+	<string name="osu_parser_error">Dificuldades tentando encontrar o erro</string>
+	<string name="require_storage_permission">O jogo não consegue rodar sem permissões de armazenamento. favor habilitar a permissão manualmente</string>
+
+	<string name="update_dialog_title">Nova versão disponivel</string>
+	<string name="update_dialog_button_confirm">Atualizar agora</string>
+	<string name="update_dialog_button_later">Depois</string>
+	<string name="update_info_checking">Procurando por atualizações, favor ser paciente</string>
+	<string name="update_info_runs_latest">Você ja esta atualizado :D</string>
+
+	<string name="test_ed_storyboard">Ver storyboards (experimental)</string>
+
+	<string name="mod_precise_is_unrank_now">O mod PRECISE não esta ranqueado por enquanto</string>
+
+
+	<!-- new data after 1.6.7(beta1) -->
+    <string name="beta_checking_upgrade">Procurando por atualizações...</string>
+	<string name="beta_check_upgrade_error">Falha ao procurar atualizações, tente novamente depois</string>
+	<string name="beta_you_are_the_latest_version">Você ja esta usando a ultima versão disponível</string>
+	<string name="beta_downloading">Baixando</string>
+	<string name="beta_click_to_view">Clique para visualizar</string>
+	<string name="beta_click_to_install">Instalar</string>
+	<string name="beta_click_to_retry">Tentar novamente</string>
+	<string name="beta_click_to_continue">Continuar</string>
+	<string name="beta_download_success">O download foi feito com sucesso</string>
+	<string name="beta_download_error">O download falhou</string>
+	<string name="beta_have_new_version">Nova versão disponivel</string>
+    <string name="menu_map_updatable">Atualizar beatmap</string>
+    <string name="message_loading_skin">Aplicando uma nova skin</string>
+
+
+	<string name="message_error_dir_not_found">Diretório não encontrado!</string>
+
+	<string name="message_update_hint">Segure para atualizar</string>
+	<string name="message_update_download_success">Atualizado com sucesso!</string>
+	<string name="message_update_download_fail">A atualização falhou tente novamente depois</string>
```

Comment:
``A atualização falhou, tente novamente mais tarde`` sounds better (and more formal) than ``tente novamente depois``

Reply:
fixed